packages/aws-cdk-lib/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -270,6 +275,10 @@ export class LustreFileSystem extends FileSystemBase {
       weeklyMaintenanceStartTime: props.lustreConfiguration.weeklyMaintenanceStartTime?.toTimestamp(),
       automaticBackupRetentionDays: props.lustreConfiguration.automaticBackupRetention?.toDays(),
       dailyAutomaticBackupStartTime: props.lustreConfiguration.dailyAutomaticBackupStartTime?.toTimestamp(),
+      driveCacheType: (
+        props.storageType === StorageType.HDD
+        && props.lustreConfiguration.deploymentType === LustreDeploymentType.PERSISTENT_1
+      ) ? 'READ' : undefined,
```

Comment:
Are there any scenarios we might want the user to set this to `NONE` instead?
I assume that users will incur higher costs if we default to `READ`.

Reply:
I've added `driveCacheType` to `LusterConfiguration` and added validation method for `driveCacheType`.

```ts
  private validateDriveCacheType(deploymentType: LustreDeploymentType, storageType?: StorageType, driveCacheType?: DriveCacheType): void {
    if (!driveCacheType) return;

    if (deploymentType !== LustreDeploymentType.PERSISTENT_1 || storageType !== StorageType.HDD) {
      throw new Error(`driveCacheType can only be set for PERSISTENT_1 HDD storage type, got: ${deploymentType} and ${storageType}`);
    }
  }
```